test/node/timestamp_tests.js

```diff
@@ -15,7 +15,9 @@ describe('Timestamp', function () {
       new BSON.Timestamp(-1, -1),
       new BSON.Timestamp(new BSON.Timestamp(0xffffffff, 0xffffffff)),
       new BSON.Timestamp(new BSON.Long(0xffffffff, 0xfffffffff, false)),
-      new BSON.Timestamp(new BSON.Long(0xffffffff, 0xfffffffff, true))
+      new BSON.Timestamp(new BSON.Long(0xffffffff, 0xfffffffff, true)),
+      new BSON.Timestamp({ t: 0xffffffff, i: 0xfffffffff }),
+      new BSON.Timestamp({ t: -1, i: -1 })
```

Comment:
You brought up a good point about Int32 and valueOf behavior in the other comment thread can we throw a quick test in to capture that:
`new BSON.Timestamp(new Int32(0xffffffff), new Int32(0xffffffff)),`

Reply:
@nbbeeken Done! (rebased to fix the linter issues)